further_link/util/copy.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+from shutil import copytree, rmtree
+
+from .user_config import default_user, get_miniscreen_projects_directory, get_gid, get_uid
+
+
+def set_directory_ownership(directory, user, recurse=True):
+    os.chown(directory, uid=get_uid(user), gid=get_gid(user))
+    for filename in os.listdir(directory):
+        file_path = os.path.join(directory, filename)
+        try:
+            if recurse and os.path.isdir(file_path):
+                set_directory_ownership(file_path, user)
+            else:
+                os.chown(file_path, uid=get_uid(user), gid=get_gid(user))
+        except Exception as e:
+            logging.error(f"Error setting '{file_path}' ownership to '{user}': {e}")
+
+
+async def do_copy_files_to_projects_directory(src_directory, directory, user=None):
+    if user is None:
+        user = default_user()
+    
+    dst_directory = get_miniscreen_projects_directory(
+        directory.get("name"), user, directory.get("username")
+    )
+    rmtree(dst_directory, ignore_errors=True)
```

Comment:
this will delete the project log file. I think this is what we want, it's best not to have a log file for out of data code, but it might be related to the issues i saw with the logfile when testing. Anyway, I don't think there are any issues here